Packages/vcs/Lib/VTKPlots.py

```diff
@@ -87,7 +90,7 @@ def endEvent(self,obj,event):
     if self.renWin is not None:
       if self.reRender:
         self.reRender = False
-        self._lastSize = None
+        #self._lastSize = None
```

Comment:
I'll give the resizing while animating a shot and see what happens; will probably have to rebuild UV-CDAT first, because mine's on a broken vistrails version.